isso/js/app/isso.js

```diff
@@ -151,7 +151,7 @@ var insert_loader = function(comment, lastcreated) {
         entrypoint = $("#isso-root");
         comment.name = 'null';
     } else {
-        entrypoint = $("#isso-" + comment.id + " > .isso-text-wrapper > .isso-follow-up");
+        entrypoint = $("#isso-" + comment.id + " > .isso-follow-up");
```

Comment:
You might want to write a test case for this - template code and `isso.js` relying on DOM selectors such as `$('#isso-x')` has caused trouble before, e.g. https://github.com/posativ/isso/issues/524

See https://isso-comments.github.io/docs/technical-docs/testing-client/ (latest docs not uploaded to posativ.org yet) and `isso/js/tests/unit/` for examples.

Reply:
I took a look at those docs and the other files, but I'm still not really sure what I should do to test it. It doesn't look like a similar scenario to the other tests.

Reply:
I'll write a test for it myself.